benchmark-script/write_operations/main.go

```diff
@@ -112,28 +113,30 @@ func runReadFileOperations() (err error) {
 
 	if err == nil {
 		fmt.Println("write benchmark completed successfully!")
-		fmt.Println("Waiting for 3 minutes")
+		fmt.Println("Waiting for 1 minutes")
 
-		time.Sleep(3 * time.Minute)
-	}
-
-	for i := 0; i < *fNumOfThreads; i++ {
-		if err = fileHandles[i].Close(); err != nil {
-			err = fmt.Errorf("while closing the fileHandle: %w", err)
-			return
-		}
+		time.Sleep(1 * time.Minute)
```

Comment:
Optional: We can remove this. I have added it to separate the metrics of two consecutive execution.